README.rst

```diff
@@ -26,21 +26,25 @@ Hardware Compatibility
 
 This release has been validated with the following hardware:
 
-* Intel® Quark™ Microcontroller SE Series.
-* Intel® Quark™ Microcontroller SE Development Platform.
-* Intel® Quark™ Microcontroller D2000 Series.
+* Intel® Quark™ SE Microcontroller C1000.
+* Intel® Quark™ SE Microcontroller C1000 Development Platform.
+* Intel® Quark™ Microcontroller D2000.
 * Intel® Quark™ Microcontroller D2000 Development Platform.
 
 External Dependencies
 *********************
 
-* The ISSM toolchain (version: 2016-05-12) is required to build the source code. It provides both the IAMCU and the ARCMCU toolchains (i586-intel-elfiamcu and arc-elf32, respectively).
```

Comment:
Why did you remove the version from here? The dependencies file does not state the version, it must be added either here or there.